client/src/cbltest/api/syncgateway.py

```diff
@@ -1527,6 +1538,154 @@ async def create_user_client(
             secure=self.secure,
         )
 
+    async def start_isgr(
```

Comment:
A small nit, instead of passing all the contents of the payload body, why not create a ISGRPayload class similar to something like PutPayload class, just keeps the code of sanitising the input inside the class and this method can purely focus on creating the ISGR